### PR TITLE
Jack/gfc 558

### PR DESCRIPTION
**Pull request by WitchellJack**

### CODE REVIEW

File: app/uk/gov/hmrc/gform/views/form/snippets/field_template_text_total.scala.html [hunk 1]
```diff
@@ -29,6 +29,18 @@
...
+    val zero = BigDecimal("0")
+    val currentAmount = BigDecimal(current)
+    if (currentAmount < zero){
+        val finalNegativeAmount = "-£" + currentAmount.abs.toString()
+        finalNegativeAmount
+    } else {
+        val finalPositiveAmount = "£" + current
+        finalPositiveAmount
+    }
+}
```
Comment (josef-vlach) [inline app/uk/gov/hmrc/gform/views/form/snippets/field_template_text_total.scala.html]: This seems to be broken in two ways.

1. We are adding `£` even for non Sterling values. (I do not see equivalent logic of removed line 53)
2. In other places of application we took approach to use `NumberFormat.getCurrencyInstance(Locale.UK)` to format Sterling values. We should use it here as well. See [TextFormatter.scala](https://github.com/hmrc/gform-frontend/blob/master/app/uk/gov/hmrc/gform/views/summary/TextFormatter.scala#L33) for inspiration.

Comment (WitchellJack) [reply app/uk/gov/hmrc/gform/views/form/snippets/field_template_text_total.scala.html]: @josef-vlach  I believe this should fix all reported issues?

**Comment by WitchellJack** [comment]
@josef-vlach This should now be finished? I have tested this in local and it works.

**Comment by josef-vlach** [comment]
Looks good. You need to rebase to resolve conflicts and please squash commits into one (no need to keep interim commits in history)

**Comment by lutnos** [comment]
Fine @josef-vlach , we normally merge Jack's commits, so if he does the rebase then. we can do the squash when we merge.  

@WitchellJack yes, please rebase, you do not need to squash, we can do that when we merge it.

**Pull request closed without merging**